manticore/platforms/evm.py

```diff
@@ -1726,16 +1746,19 @@ def __init__(self, constraints, storage=None, initial_block_number=None, initial
         self._pending_transaction = None
         self._transactions: List[Transaction] = list()
 
-        if initial_block_number is None:
-            #assume initial symbolic block
-            initial_block_number = constraints.new_bitvec(256, "BLOCKNUMBER", avoid_collisions=True)
-        self._initial_block_number = initial_block_number
-        if initial_timestamp is None:
+        if blocknumber is None:
+            #assume initial byzantium block
+            blocknumber = 4370000
```

Comment:
Don't we have this data in pyevmasm?

Reply:
https://github.com/trailofbits/pyevmasm/blob/master/pyevmasm/evmasm.py#L881-L913
we have the reverse, guess we need to add the reverse of it too :P

Reply:
So I'd say it is good to go now.

Reply:
All fork setting is not coherent. We can change the assembler, the block number and the instruction handlers are always the same (tending to byzantium).